### DISCUSSION TITLE
HP 8770W won't boot EFI file

### DISCUSSION THREAD

**Discussion by matthegg5** [SkyrilHD/HP-8x70W-Hackintosh#27] [Q&A] [answered]
Hello

I'm attempting to get Monterey installed using the latest AMD release of your EFI folder on my HP 8770w - it has AMD graphics and Intel Wifi. When I "boot from EFI file", if i choose either Opencore.efi under the OC folder, or BOOTx64.efi under the BOOT folder I get the below message.

OC: Grabbed zero system-ids for SB, this is not allowed. Halting on critical error.

I've double-checked what should be enabled and disabled on the BIOS but still the error persists. Any idea?

I have an HP 8760w I could try, maybe?

**Comment by SkyrilHD** [comment, marked as answer]
You need to disable SecureBootModel from the config.

https://github.com/SkyrilHD/HP-8x70W-Hackintosh/blob/ca175f1a48617a10b4ac7bd1fbde7b204bd25034/EFI/OC/config.plist#L1379

Change "Default" to "Disabled" (case-sensitive)

Also, there is no guarantee that 8760W will work. The focus is on 8x70W.

**Reply by matthegg5** [reply]
Thank you! Worked great, albeit on an old HDD so a bit slow... SSD on order!
